drivers/spi/spi_context.h

```diff
@@ -331,14 +338,12 @@ void spi_context_update_tx(struct spi_context *ctx, uint8_t dfs, uint32_t len)
 
 	ctx->tx_len -= len;
 	if (!ctx->tx_len) {
-		ctx->tx_count--;
-		if (ctx->tx_count) {
-			ctx->current_tx++;
-			ctx->tx_buf = (const uint8_t *)ctx->current_tx->buf;
-			ctx->tx_len = ctx->current_tx->len / dfs;
-		} else {
-			ctx->tx_buf = NULL;
-		}
+		++ctx->current_tx;
```

Comment:
Put a comment here explaining the jump (to differentiate from the spi_context_get_next_buf() call made in spi_context_buffers_setup()). Also I don't see why you change the de/in-crement syntax here (and the order of them as well)

Reply:
> Put a comment here explaining the jump (to differentiate from the spi_context_get_next_buf() call made in spi_context_buffers_setup()).

Done.

> Also I don't see why you change the de/in-crement syntax here (and the order of them as well)

I just thought it would make this more readable if both those operators were aligned on the left and these fields were updated in the same order they are passed as parameters in function calls. But it's not in any way important to me and I can reverse the order and move the operators to the right if that's preferred.